openlcs/libs/corgi.py

```diff
@@ -32,6 +36,42 @@ def __init__(self, base_url=None):
             # corgi api endpoint available in environment variable
             base_url = os.getenv("CORGI_API_PROD")
         self.base_url = base_url
+        # corgi stage support this at the moment.
+        self.default_exclude_fields = (
+            "products",
+            "product_versions",
+            "product_streams",
+            "product_variants",
+            "channels",
+        )
+        self.session = requests.Session()
+
+    def get(self, url, query_params=None, excludes=None, timeout=30,
```

Comment:
Change it to `corgi_get`? so that distinguish original`get`

Reply:
What do you mean by original `get`? since it's already in module `corgi`, I think `get` is more concise.

Reply:
such as `requests.Session().get`

Reply:
I think they don't collide as they're in different modules.

I think `corgi_get` is weird to me since it's in `corgi.py` module under `CorgiConnector` class. all functions in the class should ideally not contain any `corgi` keywords for consise.

I'd like to keep it for now, and am open to future better alternative.